### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by JoshuaPadilla** [community/community#148057] [GitHub Education]
### Hi everyone,

Hi everyone, what should I do because github student developer pack needs an University issued email address and our university stop giving us that email. Is it still possible to avail the pack or not?

**Comment by Lut-Lat-Aung** [comment]
Unfortunately, if your university no longer provides you with an official email address, obtaining the GitHub Student Developer Pack might be a bit more challenging, as GitHub typically requires a university-issued email for verification.

You can try other things

- Maybe, try contacting the Github support with your personal email to approve your student developer pack if you can show university transcript and student ID.

I hope you find my answer helpful.

**Comment by JoshuaPadilla** [comment]
Thanks, I will try contacting github support first.

**Reply by Lut-Lat-Aung** [reply]
You're welcome. Good luck.
